openrouteservice-api-tests/src/test/java/org/heigit/ors/services/routing/ResultTest.java

```diff
@@ -13,12 +13,12 @@
  */
 package org.heigit.ors.services.routing;
 
+import io.restassured.response.Response;
```

Comment:
Why is this being moved up here?

Reply:
IDE auto organize imports did reordering, it's a V1 test anyway